app/scripts/actions/index.js

```diff
@@ -303,6 +328,36 @@ export const applyWorkflowToGranule = (granuleId, workflow) => ({
   }
 });
 
+export const getCollectionByGranuleId = (dispatch, granuleId) => {
```

Comment:
Every thunk should receive `dispatch` as a parameter, so I don't think we have to pass in `dispatch` this way. I think we can change this to:

```
export const getCollectionByGranuleId = (granuleId) => {
  return (dispatch) => {
    return dispatch(getGranule(granuleId)).then((granuleResponse) => {
      const { name, version } = collectionNameVersion(granuleResponse.data.collectionId);
      return dispatch(getCollection(name, version));
    });
  }
};
```